.github/workflows/ci-build.yml

```diff
@@ -57,8 +57,10 @@ jobs:
     - name: Lint
       uses: golangci/golangci-lint-action@v2
       with:
-        version: v1.39
-        skip-go-installation: true
-        args: -E gofmt,golint,megacheck,misspell
+        version: v1.42.1
+        skip-go-installation: false
```

Comment:
is skipping the go installation intended?

Reply:
I used the same settings as in https://github.com/codeready-toolchain/sandbox-sre/blob/master/.github/workflows/workflow.yml#L66-L73 :)

Reply:
reverted in https://github.com/codeready-toolchain/host-operator/pull/552/commits/30c2070c62454a26b9179eb1d4b04f066d59d26a since we already install Go in a previous step of the same job